.travis.yml

```diff
@@ -14,7 +14,7 @@ branches:
   - /^release-\d+\.\d+\.\d+-hotfix-\d+$/
 
 env:
-  matrix:
```

Comment:
Why this change?

Reply:
Travis no longer calls this matrix, it just aliases jobs now.

Reply:
Oh okay, I wasn't able to find any docs about this, thanks!